pageserver/src/tenant/timeline.rs

```diff
@@ -996,6 +996,7 @@ impl Timeline {
     }
 
     pub(crate) const MAX_GET_VECTORED_KEYS: u64 = 32;
+    pub(crate) const VEC_GET_LAYERS_VISITED_WARN_THRESH: f64 = 800.0;
```

Comment:
Maybe put this as 512 as we don't know how much close to 1024 it is?

Reply:
Very good point: [b7c53d1](https://github.com/neondatabase/neon/pull/8228/commits/b7c53d1011ed43ccd4ae10157f755fd84a12952d)